src/app/home/work-item-widget/work-item-widget.component.ts

```diff
@@ -18,6 +18,7 @@ export class WorkItemWidgetComponent implements OnDestroy, OnInit  {
   currentSpaceId: string = 'default';
   loading: boolean = false;
   loggedInUser: User;
+  loggedInUserId: string;
```

Comment:
This looks redundant to me because `loggedInUser` is now no longer used for anything. It would make more sense to me if we used the `loggedInUser.id` value elsewhere in the component as opposed to subscribing to the `UserService` to retrieve only an `id` value.

There seems to be a problem in the `.spec` file hooking up the `fakeUser` to the mocked user service .. removing the `publish()` here [[0]](https://github.com/fabric8-ui/fabric8-ui/blob/master/src/app/home/work-item-widget/work-item-widget.component.spec.ts#L96) seems to rectify the situation, and we can use the `fakeUser` observable as a connectable observable.

[0] https://github.com/fabric8-ui/fabric8-ui/blob/master/src/app/home/work-item-widget/work-item-widget.component.spec.ts#L96